### PR TITLE
User model

### PR DESCRIPTION
**Pull request by bhflm**
## Summary

fixed the previous commit, now done with the right language, 
added the new user model for the db
fixed some typos w/ camel case
## Known Issues

no issues yet

## Trello Card

https://trello.com/c/64MI9Yr8/27-modelo-de-usuario

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const errors = require('../errors');
+
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define(
```
Comment (gonziesc) [inline app/models/user.js]: plural

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    'user',
```
Comment (gonziesc) [inline app/models/user.js]: same

File: migrations/migrations/20180917134433-create-users.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('usernames', {
```
Comment (gonziesc) [inline migrations/migrations/20180917134433-create-users.js]: change to users here

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: app/models/user.js (hunk 1 above)
Comment (bhflm) [reply app/models/user.js]: done in ce1e674d2b13fc863c46dfcbc76348e5325c23b7

File: app/models/user.js (hunk 2 above)
Comment (bhflm) [reply app/models/user.js]: done here ce1e674d2b13fc863c46dfcbc76348e5325c23b7

File: migrations/migrations/20180917134433-create-users.js (hunk 3 above)
Comment (bhflm) [reply migrations/migrations/20180917134433-create-users.js]: ce1e674d2b13fc863c46dfcbc76348e5325c23b7

**Review by gonziesc — APPROVED** [review]

**Pull request merged**